src/ring_theory/adjoin_root.lean

```diff
@@ -8,6 +8,27 @@ Adjoining roots of polynomials
 
 import data.polynomial ring_theory.principal_ideal_domain
 
+/-!
+# Adjoining roots of polynomials
+
+This file defines the commutative ring `adjoin_root f`, the ring R[X]/(f) obtained from a
+commutative ring R and a polynomial f : R[X]. If furthermore `R` is a field and `f` is
```

Comment:
```suggestion
This file defines the commutative ring `adjoin_root f`, the ring $R[X]/(f)$ obtained from a
commutative ring `R` and a polynomial `f : R[X]`. If furthermore `R` is a field and `f` is
```
Not sure if LaTeX helps or hurts...

Reply:
My impression was that the general idea was to use unicode not LaTeX, but I definitely agree with the quotes.